pkg/apis/batch/v1/conversion.go

```diff
@@ -26,21 +26,36 @@ import (
 )
 
 func addConversionFuncs(scheme *runtime.Scheme) error {
-	return scheme.AddFieldLabelConversionFunc(SchemeGroupVersion.WithKind("Job"),
+	// Add field label conversions for kinds having selectable nothing but ObjectMeta fields.
```

Comment:
comment seems inaccurate: Job has `status.successful`.

nit: move the comment above the function definition, but perhaps its redundant?

Reply:
Dropped